src/main/scala/de/heikoseeberger/sbtfresh/Template.scala

```diff
@@ -252,7 +260,10 @@ private object Template {
        |addSbtPlugin("de.heikoseeberger" % "sbt-header"   % "1.6.0")
        |""".stripMargin
 
-  def readme(name: String): String =
+  def readme(name: String, license: String): String = {
+    val licenseKind: License.EnumVal = getLicenseKind(license)
```

Comment:
Do we need this assignment? Maybe just call `getLicensKind(license)` inline in the next line.